### DISCUSSION TITLE
Setting up a relay

### DISCUSSION THREAD

**Discussion by grantrules** [easytarget/esp32-cam-webserver#268] [Q&A]
Since these things can only handle one streaming connection or so, I've been trying to find a way to set up a repeater on a different computer. Has anyone had success? I've tried with VLC and https://github.com/blueimp/mjpeg-server but no success.

**Comment by daschr** [comment]
You can checkout the multi-stream branch which supports multiple connections: https://github.com/easytarget/esp32-cam-webserver/tree/multi-stream 
I'm using it for months. I think @easytarget will merge it soon to the main branch.

**Comment by easytarget** [comment]
Sorry; I am not doing any further development on this project. it is old code and somewhat obsolete. I am currently working on retiring it; I will try to link to a more actively updated fork when I do.
